core/src/main/scala/kafka/metrics/clientmetrics/CmClientInstanceState.scala

```diff
@@ -14,77 +31,80 @@ import scala.collection.mutable.ListBuffer
-    val targetGroups = new java.util.ArrayList[SubscriptionGroup]()
+    val targetSubscriptions = new java.util.ArrayList[SubscriptionInfo]()
     var allMetricsSubscribed = false
 
-    sgroups.forEach(v =>
+    subscriptions.forEach(v =>
       if (clientInfo.isMatched(v.getClientMatchingPatterns)) {
         allMetricsSubscribed = allMetricsSubscribed | v.getAllMetricsSubscribed
         targetMetrics = targetMetrics ++ v.getSubscribedMetrics
-        targetGroups.add(v)
+        targetSubscriptions.add(v)
         pushInterval = Math.min(pushInterval, v.getPushIntervalMs)
       }
     )
 
-    // if pushinterval == 0 means, metrics collection is disabled for this client so clear all the metrics and just
-    // send the empty metrics list to the client.
-    // Otherwise, if client matches with any group that has the property `allMetricsSubscribed` which means there is
-    // no need for filtering the metrics, so as per KIP-714 protocol just send the empty string as the contents
-    // of the list so that client would send all the metrics updates Otherwise, just use the compiled metrics.
+    // if pushInterval == 0 means, metrics collection is disabled for this client so clear all the metrics
+    // and just send the empty metrics list to the client.
+    // Otherwise, if client matches with any subscription that has the property `allMetricsSubscribed`
+    // which means there is no need for filtering the metrics, so as per KIP-714 protocol just send the
+    // empty string as the contents of the list so that client would send all the metrics updates
+    // Otherwise, just use the compiled metrics.
     if (pushInterval == 0) {
+      info(s"Metrics collection is disabled for the client: ${id.toString}")
       targetMetrics.clear()
     } else if (allMetricsSubscribed) {
       targetMetrics.clear()
       targetMetrics.append("")
     }
 
-    new CmClientInstanceState(id,  clientInfo, targetGroups, targetMetrics.toList, pushInterval, allMetricsSubscribed)
+    new CmClientInstanceState(id,  clientInfo, targetSubscriptions,
+                              targetMetrics.toList, pushInterval, allMetricsSubscribed)
   }
 
 }
 
 class CmClientInstanceState private (clientInstanceId: Uuid,
                                      clientInfo: CmClientInformation,
-                                     sgroups: java.util.Collection[SubscriptionGroup],
+                                     subscriptions: java.util.Collection[SubscriptionInfo],
                                      var metrics: List[String],
                                      pushIntervalMs: Int,
                                      allMetricsSubscribed: Boolean) {
 
-  private val metricsReceivedTs = Calendar.getInstance.getTime
+  private val lastAccessTS = Calendar.getInstance.getTime
   private val subscriptionId = computeSubscriptionId
 
   def getPushIntervalMs = pushIntervalMs
-  def getLastMetricsReceivedTs = metricsReceivedTs
+  def getLastAccessTS = lastAccessTS
```

Comment:
not sure why using upper case TS here, typically we use Ts no?

Reply:
There is no specific reason, will change it to Ts